cmd/keytransparency-signer/backend.go

```diff
@@ -132,7 +133,11 @@ func main() {
 	}
 	mutator := entry.New()
 
+	if *maxEpochDuration < *minEpochDuration {
+		glog.Exitf("Maximum time between epoch creation (%v) must be grater or equal minimum time between epoch creation.")
```

Comment:
Consider following the canonical "got, want" style for error messages. 
https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures